ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/JTreeFixture.kt

```diff
@@ -12,14 +12,32 @@ class JTreeFixture(
     remoteRobot: RemoteRobot,
     remoteComponent: RemoteComponent
 ) : ComponentFixture(remoteRobot, remoteComponent) {
+    fun clickPath(vararg paths: String) = runPath("clickPath", *paths)
+    fun expandPath(vararg paths: String) = runPath("expandPath", *paths)
+    fun rightClickPath(vararg paths: String) = runPath("rightClickPath", *paths)
+    fun doubleClickPath(vararg paths: String) = runPath("doubleClickPath", *paths)
```

Comment:
runJsMethod? runPath is confusing till I saw the actual impl

Reply:
Changed to runJsPathMethod